internal/testbed/integration/kafka/e2e_test.go

```diff
@@ -273,25 +271,57 @@ func TestE2E_Kafka(t *testing.T) {
 		ptracetest.IgnoreEndTimestamp(),
 		ptracetest.IgnoreTraceID(),
 		ptracetest.IgnoreSpanID(),
-		ptracetest.IgnoreEndTimestamp(),
 		ptracetest.IgnoreSpansOrder(),
 	}
 
 	expectedTraces, err := golden.ReadTraces(expectedTracesFile)
 	require.NoError(t, err)
 
 	require.EventuallyWithT(t, func(tt *assert.CollectT) {
-		gotTraces := tracesConsumer.AllTraces()[len(tracesConsumer.AllTraces())-1]
+		all := tracesConsumer.AllTraces()
+		require.NotEmpty(tt, all)
+		got := all[len(all)-1]
 		testutil.MaskParentSpanID(expectedTraces)
-		testutil.MaskParentSpanID(gotTraces)
-		assert.NoError(tt,
-			ptracetest.CompareTraces(
-				expectedTraces,
-				gotTraces,
-				traceCompareOptions...,
-			),
-		)
-	}, 3*time.Minute, 1*time.Second)
+		testutil.MaskParentSpanID(got)
+		assert.NoError(tt, ptracetest.CompareTraces(expectedTraces, got, traceCompareOptions...))
+	}, compareTimeout, compareTraceTick)
 
 	t.Logf("Traces checked successfully")
 }
+
+func kubeconfigFromEnvOrDefault() string {
```

Comment:
these helper functions seem like they could be useful in the other tests as well. would it make sense to move them into a separate package?